site/profile/templates/it/graylog_ssl_cfg.epp

```diff
@@ -0,0 +1,23 @@
+[req]
```

Comment:
there are modules to generate self-sign certs; no need to provide all the parameters